test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,58 @@
+./test_scripts/API/RAI/001_ATF_N_HMI_Does_Not_Respond_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_HMI_capabilities_HMI_doesn't_respond_UI.GetCapabilities

```

Comment:
@aderiabin, @OVikhrov, I am not agree with you, because SDL is tested like a Black Box and you can not be sure how each error of message is processed by SDL and how this will affect to SDL behaviour and current CRQ. 
Also each script is running about 30 sec and this will not affect with many time for execution.

For regression testing additional scripts are added in scope of RAI with UI.Isready missing or available: false.
See b74b1f9410076d9caea0a1b3f4dce862f4fa8182